src/consent-manager/index.tsx

```diff
@@ -24,7 +24,8 @@ export default class ConsentManager extends PureComponent<ConsentManagerProps, {
     bannerSubContent: 'You can change your preferences at any time.',
     bannerBackgroundColor: '#1f4160',
     preferencesDialogTitle: 'Website Data Collection Preferences',
-    cancelDialogTitle: 'Are you sure you want to cancel?'
+    cancelDialogTitle: 'Are you sure you want to cancel?',
+    defaultDestinationBehavior: 'disable'
```

Comment:
This prop is defaulted to 'disable' so as to not automatically change behavior for users updating their consent manager version.